libs/pika/resource_partitioner/examples/oversubscribing_resource_partitioner.cpp

```diff
@@ -86,18 +86,18 @@ int pika_main(/*pika::program_options::variables_map& vm*/)
         pika::execution::thread_priority::critical);
     pika::execution::parallel_executor normal_priority_executor;
 
-    pika::execution::parallel_executor mpi_executor;
+    pika::execution::parallel_executor pool_executor;
```

Comment:
These changes probably don't belong in this PR?

Reply:
no, but they are flyby fixes with "mpi" in them so I thought why-not.

Reply:
Ah, right. I missed that it indeed doesn't make sense to call it `mpi_executor`. Fine, let's keep it here (same for the other one).